config/manager/manager.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
```

Comment:
I am not a fan of the generated value `controller-manager` that is used all throughout the manifests. I prefer to change it to the name of the operator. In this case I would replace `controller-manager` with `cass-operator`. Thoughts?

Reply:
We need to create a script then that modifies them after every time controller-gen is ran (manifest / generate), since the kubebuilder will always use that name (no configuration option).

Reply:
hmm... I tried the following:

* Add field to CassandraDatacenterSpec
* Modify RBAC annotation in cassandradatacenter_controller.go
* In manager.yaml replace controller-manager with with cass-operator
* Run `make manifests`

My changes in deployment.yaml stayed intact.

Reply:
And your deployment has a funny name afterwards also ;)

```
NAME                                          READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/cass-operator-cass-operator   0/1     1            0           114s

NAME                                             READY   STATUS             RESTARTS   AGE
pod/cass-operator-cass-operator-b687684f-wc8bn   1/2     CrashLoopBackOff   2          114s

NAME                                                   DESIRED   CURRENT   READY   AGE
replicaset.apps/cass-operator-cass-operator-b687684f   1         1         0       114s
```

The name-prefix in the kustomize template hides most of the controller-manager instances. The label isn't modified, but all the names are. We might want to look at what to do with the control-plane, it has some effect on AKS at least..

Reply:
My comments were based off of what I have done elsewhere. I failed to mention that I had removed the prefix. If there are changes we want to make around the kustomize stuff, they can probably be done in follow up PRs. The tests are passing which is the biggest thing.